lib/vsc/install/shared_setup.py

```diff
@@ -315,7 +316,10 @@ def reload_vsc_modules(self):
 
         # reimport
         for name in reload_vsc_modules:
-            __import__(name)
+            try:
+                __import__(name)
+            except ImportError:
+                log.error('Failed to reload/import %s (sys.path %s). Continuing, fingers crossed.' % (name, sys.path))
```

Comment:
hmm... :-)

shouldn't we prefer a hard crash here?

how do we know we're still testing what we're supposed to be testing if this fails?